src/components/categories/Categories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Categories = () => (
+  <>
+  </>
+);
```

Comment:
- [ ] Following the project requirements, Categories should display only a button with the text "Check status".